charts/smokey/templates/cronjob.yaml

```diff
@@ -26,13 +28,9 @@ spec:
           - emptyDir: {}
             name: tmp
           containers:
-          - args:
-            - cucumber
-            - --profile={{ .Values.govukEnvironment }}
-            - --strict-undefined
-            - --tags=not @notreplatforming and not @not{{ .Values.govukEnvironment }}
+          - args: {{ .args }}
```

Comment:
You're defining `.args` as a list in `values.yaml`, but interpreting it as a string here (which makes it harder to see what's going on). (You probably want a `toYaml` here somewhere.)